app/src/main/java/ru/prometeydev/movie/model/MoviesRemoteMediator.kt

```diff
@@ -33,17 +31,13 @@ class MoviesRemoteMediator(
 
         return@withContext try {
 
-            val loadImageConfig = async {
-                if (baseImageUrl.isEmpty()) {
-                    baseImageUrl = api.getConfiguration().images.secureBaseUrl
-                }
+            val jobBaseImageUrl = launch {
+                baseImageUrl = getImagesBaseUrl()
```

Comment:
Mm, so baseUrl will be assigned to itself after the first load, why not the first variant and maybe extracted to function?